Sources/CardPayments/CardClientError.swift

```diff
@@ -39,7 +39,7 @@ enum CardClientError {
         case malformedDeeplinkURLError
 
         /// 10. Cancellation from 3DS verification
-        case threeDSCancellation
+        case canceledError
```

Comment:
nit: For this error `threeDSecureCanceled` makes sense to me, since CardClient won't always perform 3DS verification. It like how it implicitly implies that 3DS was attempted.

Reply:
I thought about this and I was opting for brevity instead of the error name being more descriptive.
The only way card processing can be canceled is in 3DS. I wanted to keep it as simple as possible 
since cancellation as an error is new.

But I see your point that for non 3DS flow, it would not be clear that this is not an error that's only for 3DS.